addon/components/rdf-input-fields/heading.js

```diff
@@ -0,0 +1,7 @@
+import Component from '@glimmer/component';
+
+export default class RdfHeadingComponent extends Component {
+  get titleSkin() {
+    return this.args.field.options.skin || '6';
```

Comment:
I would use the same defaults as the AuHeading component so just return the option value here maybe? That way it will use the AuHeading default value if it's undefined. It requires adding the config for your use-case, but I think it's a better long term default value.

Reply:
@Windvis if you only add `@level` argument, nothing will change to the skin, shouldn't the `@skin` argument be related to the level if skin is not passed?

Reply:
Yea, that's a valid point. That's more a question for Appuniversum though 😄.